### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by C47D**
Fix examples compilation (might not be rusty enough):
- arc
- animation
- bar. I had to comment out `lvgl::event_send` to make this example compile and run.
- button_click

Needs #177

### CODE REVIEW

**Comment by C47D** [comment]
Can you help me reviewing this @AlixANNERAUD?

**Review by AlixANNERAUD — COMMENTED** [review]
LGTM, maybe the 1500 ms is too much.

File: examples/bar.rs [hunk 1]
```diff
@@ -47,24 +47,27 @@ fn main() -> Result<(), LvError> {
...
+    loading_lbl.add_style(Part::Main, &mut loading_style);
 
     let mut i = 0;
     'running: loop {
         let start = Instant::now();
         if i > 100 {
             i = 0;
-            lvgl::event_send(&mut bar, Event::Clicked)?;
+            // Enabling this line gives the followong errors:
+            // - error[E0597]: `ind_style` does not live long enough when adding an style to the bar
```
[the remark this answers is not in the document]
Comment (C47D) [reply examples/bar.rs:66]: I will add it into #184

File: examples/bar.rs [hunk 2]
```diff
@@ -47,24 +47,27 @@ fn main() -> Result<(), LvError> {
...
+            // - implementation of `Widget` is not general enough
```
[the remark this answers is not in the document]
Comment (C47D) [reply examples/bar.rs:67]: I will add it into #184

File: examples/bar.rs [hunk 3]
```diff
@@ -76,7 +79,7 @@ fn main() -> Result<(), LvError> {
                 _ => {}
             }
         }
-        sleep(Duration::from_millis(15));
+        sleep(Duration::from_millis(1500));
```
[the remark this answers is not in the document]
Comment (C47D) [reply examples/bar.rs]: Done in newest commit

**Review by AlixANNERAUD — APPROVED** [review]

**Review by kisvegabor — APPROVED** [review]

**Pull request merged**